src/main/java/com/metamx/emitter/core/LoggingEmitter.java

```diff
@@ -86,11 +96,31 @@ public void start()
   @Override
   public void emit(Event event)
   {
-    synchronized (started) {
-      if (!started.get()) {
-        throw new RejectedExecutionException("Service not started.");
-      }
+    if (!started.get()) {
+      throw new RejectedExecutionException("Service not started.");
+    }
+
+    switch (eventsToLog) {
```

Comment:
Events to log never changes. Can the constructor simply make a Predicate which is checked here?